.github/CODEOWNERS

```diff
@@ -9,10 +9,11 @@
 # explicitly taken by someone else.
 *                               @googleapis/yoshi-go-admins
 
-/bigtable/       @googleapis/api-bigtable @googleapis/yoshi-go-admins
-/datastore/      @googleapis/firestore-dpe @googleapis/yoshi-go-admins
-/errorreporting/ @GoogleCloudPlatform/observability-devx @googleapis/yoshi-go-admins @nicoleczhu
-/firestore/      @googleapis/firestore-dpe @googleapis/yoshi-go-admins
-/logging/        @GoogleCloudPlatform/observability-devx @googleapis/yoshi-go-admins @nicoleczhu
-/storage/        @GoogleCloudPlatform/storage-dpe @googleapis/yoshi-go-admins
-/iot/            @gcseh @GoogleCloudPlatform/api-iot @googleapis/yoshi-go-admins
+/bigtable/**/*.go                     @GoogleCloudPlatform/cloud-native-db-dpes
```

Comment:
Ah, this looks new. Why restrict to Go files? There are also READMEs and other files that you probably want to own?

Reply:
This is unchanged from the last time you reviewed. [It's matching the pattern in other repos](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/1f9501c6de15130401b57878e28d051b17d1ec01/.github/CODEOWNERS#L18) to only target source files, to avoid requiring our review for Renovate updates.

Reply:
I think that will leave out some important stuff. And the root owner (not here now, but will be added later) should be handling Renovate updates, avoiding the need for individual team approvals.

For example, check out https://github.com/GoogleCloudPlatform/golang-samples/tree/master/cloudsql/mysql/database-sql. There are many non-`.go` files in there the Cloud SQL team owns.